helm/stunner-gateway-operator/values.yaml

```
@@ -48,6 +48,8 @@ stunnerGatewayOperator:
         name: docker.io/l7mp/stunnerd
         pullPolicy: IfNotPresent
         tag: 1.0.0
+      imagePullSecrets:
+        - name: image-pull-secret
```

Please comment out the example value
```suggestion
      imagePullSecrets:
        # - name: image-pull-secret
```